### ISSUE TITLE
It does not work anymore

### ISSUE THREAD

**Issue by marste** [mathaywarduk/jekyll-search#16]
Here's the full error we're seeing:

[33m    Liquid Warning: Liquid syntax error (line 5): Expected id but found open_square in "item in site.[item.label]" in feeds/feed.json[0m
[31m  Liquid Exception: nil is not a symbol nor a string in feeds/feed.json[0m
[31m             Fatal: TypeError[0m

It looks like the error is caused by the syntax of this line in your feeds.json

**Comment by philwareham** [comment]
Yep, same for me. The latest GitHub Jekyll (v3.5) refuses to build because of the feeds.json validation error. https://github.com/textpattern/textpattern.github.io/issues/77#issuecomment-322315041

**Comment by philwareham** [comment]
Changing line 7 of `feeds/json.feed` from:

    {% for item in site.[item.label] %}

to:

    {% for item in site.item.label %}

Allows Jekyll to build and *on my site* search now works as before. However, since I don't use 'collections' on my Jekyll site I can't guarantee that the above is a fully working fix for everyone.

**Comment by jeffleeder** [comment]
had this issue changed it to  {% for post in site.categories['categoryname'] %}  this worked
